pom.xml

```diff
@@ -91,7 +91,6 @@
     <dependency>
       <groupId>com.github.spotbugs</groupId>
       <artifactId>spotbugs-annotations</artifactId>
-      <version>4.5.1</version>
```

Comment:
was previously overriding a managed version (with either the same or a lower version)